src/main/cpp/RLBotInterface/RLBotMessages/FileMappings/FileMappings.cpp

```diff
@@ -51,8 +51,7 @@ namespace FileMappings
 
 	void Lock(void* pSharedMem)
 	{
-		while (InterlockedCompareExchange((unsigned int*)pSharedMem, LOCKED, UNLOCKED) != UNLOCKED)
-			Sleep(0);
+		while (InterlockedCompareExchange((unsigned int*)pSharedMem, LOCKED, UNLOCKED) != UNLOCKED) {}
```

Comment:
The sleep was probably significant: https://stackoverflow.com/questions/3727420/significance-of-sleep0

Moot point though, this file is gone in 1.15+ :)